Database/Procedures/2020_12_14_007_SP_SetRoom.sql

```diff
@@ -0,0 +1,29 @@
+USE [Hotels_DB]
+GO
+/****** Object:  StoredProcedure [dbo].[SetRoom]    Script Date: 14.12.2020 12:35:57 ******/
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+-- =============================================
+-- Author:		<Author,,Name>
+-- Create date: <Create Date,,>
+-- Description:	<Description,,>
+-- =============================================
+CREATE PROCEDURE [dbo].[SetRoom] 
+@Price int,
+@RoomNumber int,
+@TV int,

```

Comment:
Same question from above

Reply:
Agree with u, TV have to be bit type. Will change.